### ISSUE TITLE
Idea: Button that passes the filtered list to a list parameter in a microflow/nanoflow

### ISSUE THREAD

**Issue by tmisker** [mendixlabs/list-view-controls#34]
When working with list view controls, I have no idea in my application what the user has filtered on. If I want to pass the complete list that the user sees to a microflow (/nanoflow), I have to build custom search options (first the populate the list, later to pass the list to a flow).
A widget that creates a button that will pass the filtered list to a flow, would be nice.

**Comment by tmisker** [comment]
Exactly! The constraints and perhaps sorting of the various list view controls combined is the input for the microflow parameter.

**Comment by sjorsschultz** [comment]
+1 for me. This would be a really nice feature!
